test/integration/test_model_output_integration.py

```diff
@@ -73,7 +72,7 @@ def test_missing_model_output_id_mixture(tmpdir, test_file_path):
     assert len(null_output_type_rows) == 8
 
 
-def test_model_output_csv_schema(tmpdir, test_file_path, expected_model_output_schema):
+def test_model_output_csv_schema(tmpdir, test_file_path, expected_model_output_schema, schema_origin_date):
     """Test the parquet schema on files written by ModelOutputHandler."""
```

Comment:
Great! So this is effectively the test I asked for right? Which is now using an actual schema? And because we've tested that `_read_tasks` and  `_get_schema` work as expected, we can be confident that when not mocked, the integrated behaviour of the functions will work as expected, right?